cli/Valet/Nginx.php

```diff
@@ -102,6 +103,20 @@ function installNginxDirectory()
     }
 
     /**
+     * Check nginx.conf for errors.
+     */
+    private function lint()
+    {
+        $this->cli->quietly(
+            'sudo nginx -c '.static::NGINX_CONF.' -t',
+            function() use (&$error) {
+                info('Nginx cannot start, please check your nginx.conf.');
+                die;
```

Comment:
Instead of calling info and dying here, throw a DomainException like we do elsewhere:

https://github.com/laravel/valet/blob/master/cli/Valet/Brew.php#L103